sdk/client-core/core/src/main/java/io/clientcore/core/util/ServerSentEventUtils.java

```diff
@@ -1,14 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package io.clientcore.core.implementation.util;
+package io.clientcore.core.util;
```

Comment:
We should also update the module-info to remove this being exported.

Reply:
We did :)